package-lock.json

```diff
@@ -4041,8 +4047,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
```

Comment:
your package lock shouldnt have removed the "optional" tags from it

Reply:
ive only run 'npm ci', its just to do with npm versions